src/composables/auth.ts

```diff
@@ -153,16 +154,19 @@ export const useAuth = createCustomScopedComposable(() => {
     const sessionEncryptionKey = await getSessionEncryptionKey();
     if (sessionEncryptionKey) {
       setEncryptionKey(sessionEncryptionKey);
+      isAuthenticated.value = true;
```

Comment:
Since we are restoring the session here we should also set `isAuthenticated` to true